dockerfiles/postgres/backup.sh

```diff
@@ -1,30 +0,0 @@
-#!/bin/bash
```

Comment:
I removed this with the assumption that we don't use these anymore. I can revert the changes if these are in-use (Though we probably need to use a custom Docker image then).

C.C: @jefftriplett